pkgs/by-name/so/sollya/package.nix

```diff
@@ -26,6 +26,10 @@ stdenv.mkDerivation rec {
     fplll
   ];
 
+  configureFlags = [
+    "--with-xml2-config=${lib.getExe' (lib.getDev libxml2) "xml2-config"}"
+  ];
```

Comment:
```suggestion
  nativeBuildInputs = [
    libxml2
  ];
```

Does this work?

Reply:
It does, but it will give you the wrong output. For example the native `xml2-config --libs`: `-L/nix/store/shg1vgcjxghqxhl0sifwgr0g2zg6bbv7-libxml2-2.13.4/lib`